nengo/config.py

```diff
@@ -84,6 +84,10 @@ def __repr__(self):
         return "<%s[%s]{%s}>" % (type(self).__name__,
                                  self._configures.__name__, ", ".join(params))
 
+    def __setstate__(self, state):
+        for k, v in state.items():
+            setattr(self, k, v)
```

Comment:
Hm, I was pretty sure that there is no `__getattr__` on the superclass... so what's happening is it's raising the attribute error, which `pickle` takes to mean that `__setstate__` doesn't exist. So yeah, I would avoid that -- it's another result of the asymmetry between `__setattr__` and `__getattr__` (there's more symmetry with `__setattr__` and `__getattribute__`)